src/playwright/testplan.ts

```
@@ -1,29 +1,31 @@
   templates: string[];
-  tssc: {
-    git: GitType;
-    ci: CIType;
-    registry: ImageRegistryType;
-    tpa: string;
-    acs: string;
-  };
+  tsscConfigs: TSScConfig[];
   tests: string[];
 
   constructor(data: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)